lib/src/widgets/overlay.dart

```diff
@@ -179,12 +203,16 @@ class _DescribedFeatureOverlayState extends State<DescribedFeatureOverlay>
     });
   }
 
+  String activeFeatureId;
```

Comment:
This doesn't seem like a good idea, as this reintroduces a dependency towards the bloc's logic

Reply:
I did not see that `Bloc` saves this as well. Should I use that instead?

Reply:
I replaced this by a bool variable named `_isShowing`, tell me what you think.